content/en/blog/_posts/2024-09-20-kubeadm-v1beta4.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: 'Kubernetes v1.31: kubeadm v1beta4'
+date: 2024-08-23
+slug: kubernetes-1-31-kubeadm-v1beta4
+author: >
+   Paco Xu (DaoCloud)
+---
+
+The [`kubeadm`](/docs/reference/setup-tools/kubeadm/) configuration
+([v1beta4](/docs/reference/config-api/kubeadm-config.v1beta4/))
+is released with Kubernetes v1.31. v1beta3 is still supported,
+but is marked as deprecated. It will be removed after a minimum
+of 3 Kubernetes releases.
+
+This guide will walk you through key changes and update about the kubeadm v1beta4 configuration,
+and how to migrate from v1beta3 to v1beta4.
+
+Details about v1beta4 can be found in the
+[API doc]((/docs/reference/config-api/kubeadm-config.v1beta4/)).
+The documentation is maintained in the
+[code](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go)
+and generated for the kubernetes.io website.
```

Comment:
I don't think readers will want to know this detail.
```suggestion
```

Reply:
OK to remove this. But for kube contributors, this is still a thing that they should know.